packages/main/src/Toolbar.hbs

```diff
@@ -0,0 +1,22 @@
+<div
+	class="ui5-tb-root"
+	tabindex="{{tabIndex}}"
+	part="otb"
```

Comment:
normally I would associate the notion of `otb` with the toolbar itself (the custom element), and this CSS part I would consider `root`